pkg/api-server/coordinates_ws.go

```diff
@@ -0,0 +1,16 @@
+package api_server
+
+import (
+	kuma_cp "github.com/Kong/kuma/pkg/config/app/kuma-cp"
+	"github.com/Kong/kuma/pkg/coordinates"
+	"github.com/emicklei/go-restful"
+)
+
+func componentsWs(cfg kuma_cp.Config) *restful.WebService {
```

Comment:
I don't like the fact that `api-server` now depends on configuration of the entire `Control Plane`.

It makes it seem that `API Server` is inseparable from the `Control Plane`.

I would suggest an alternative solution:
* make configuration of available APIs part of `ApiServerConfig` (effectively, move existing `coordinates.Coordinates` types into `pkg/config/api-server/catalogue`)
* don't map this configuration to YAML or environment variables yet (we reserve this feature for the future but don't want to expose it to the users right away)
* auto-configure  available APIs at `autoconfigure` stage of the `bootstrap` process (effectively, move logic from `pkg/coordinates/types.go` into `pkg/core/bootstrap/autoconfigure.go`)